test/em_real/examples.namelist

```diff
@@ -56,6 +56,23 @@ Note, this is not a namelist.input file. Find what interests you, and cut and pa
  opt_stc  = 1,
 /
 
+** Using BEP urban  model
+&physics
```

Comment:
@smileMchen 
Ming,
Make this more general. Have this be the Urban Options. For the UCM scheme, include sf_urban_physics = 1. For BEP and BEM (which you already have), would you also add the lines to include those options: sf_urban_physics = 2 or 3.